src/components/mobile-offer-header/MobileOfferHeader.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="mobile-offer-header">
```

Comment:
As in the previous PR, if this is a header, shouldn't we use the `header` html element to be semantically correct?